src/OrchardCore.Modules/OrchardCore.Media/Views/Admin/MediaApplication.cshtml

```diff
@@ -41,7 +41,7 @@
                             @T["Invert"]
                         </a>
                         <a href="javascript:;" class="btn btn-light btn-sm me-2" v-on:click="deleteMediaList" :class="{disabled: selectedMedias.length < 1 }">
-                            @T["Delete"] <span class="badge rounded-pill bg-light" v-show="selectedMedias.length > 0">{{ selectedMedias.length}}</span>
+                            @T["Delete"] <span class="badge rounded-pill text-bg-light" v-show="selectedMedias.length > 0">{{ selectedMedias.length}}</span>
```

Comment:
@agriffard does this look good on a dark theme? If not, maybe try text-bg-body or our custom text-by-theme

Reply:
text-bg-light:
![image](https://github.com/OrchardCMS/OrchardCore/assets/703248/4f8c2d1a-abf1-4e99-928d-e6ea896ddff3)

text-bg-body does not add anything.

text-by-theme:
![image](https://github.com/OrchardCMS/OrchardCore/assets/703248/2155efaa-6acd-45cc-9b33-493382b5ac29)

Reply:
@agriffard try using `text-bg-theme` again please. Also, I would change the buttons from `bg-light` to `bg-secondary` and I would suggest removing the `me-2` class from the buttons. So the buttons will have these classes instead `btn btn-secodary btn-sm`

Reply:
`btn-secondary` and `text-bg-theme` seem to do the job:

![image](https://github.com/OrchardCMS/OrchardCore/assets/703248/3d8f9bd2-10bb-45c5-9dfd-2cc4508e6044)

![image](https://github.com/OrchardCMS/OrchardCore/assets/703248/f8021eef-5280-4e9a-b4e4-5bb55431a860)

Reply:
Beautiful! Thank you!